src/metrics/include/metrics/Period.hpp

```diff
@@ -21,6 +21,8 @@
 
 namespace genny::metrics {
 
+const auto NANO_PER_SECOND = 1000000000;
```

Comment:
shouldn't need this - `std::chrono` can do all this math using `std::chrono::duration_cast`.